js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/graphModals/DeleteEdgeModal.tsx

```diff
@@ -40,7 +39,7 @@ const useDeleteEdgeAction = (selectedAction?: SelectedActionType) => {
 };
 export const DeleteEdgeModal = () => {
   const { selectedAction, onClearAction } = useGraph();
-  const { edgeId, edgeData, deleteEdge } = useDeleteEdgeAction(selectedAction);
+  const { edgeId, deleteEdge } = useDeleteEdgeAction(selectedAction);
```

Comment:
why was this removed?

Reply:
this was removed because old viewer also doesn't have this, and it can look very messy with a lot of attributes